src/main/java/core/model/player/IMatchRoleID.java

```diff
@@ -56,8 +55,8 @@
     int substXT2 = 213; // 	Backup (Extra)
     List<Integer> aBackupssMatchRoleID = IntStream.rangeClosed(207, 213).boxed().collect(Collectors.toList());
     List<Integer> aSubsAndBackupssMatchRoleID = IntStream.rangeClosed(200, 213).boxed().collect(Collectors.toList());
```

Comment:
(I know you didn't touch those, but I don't quite understand why these are so complex, this should do:

```
    List<Integer> aBackupssMatchRoleID = Arrays.asList(207, 213);
    List<Integer> aSubsAndBackupssMatchRoleID = Arrays.asList(200, 213);
```
or even with `List.of` like below)